pom.xml

```diff
@@ -654,6 +654,13 @@
 					</exclusion>
 				</exclusions>
 			</dependency>
+			<!-- Upgrade commons-compress because of vulnerability; transitive dep of testcontainers 1.19.x -->
+			<dependency>
+				<groupId>org.apache.commons</groupId>
+				<artifactId>commons-compress</artifactId>
+				<version>1.27.1</version>
+				<scope>test</scope>
```

Comment:
```suggestion
```
Ik denk dat je beter de default scope op compile kan laten staan, je overschrijft hem in de core en test poms toch al.

Reply:
Ah ja goed gespot